docs/man_pages/project/creation/init.md

```diff
@@ -0,0 +1,20 @@
+init
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns init [--path <Directory>] [--force]`
+
+Initializes a project for development. It will ask you interactively to configure the project and will write a package.json for you.
+
+### Options
+* `--path` - Specifies the directory where you want to initialize the project, if different from the current directory. The directory must be empty.
+* `--force` - Will use only defaults and not prompt you for any options.
```

Comment:
If set, applies the default project configuration and does not show the interactive prompt. The default project configuration targets the latest official runtimes and sets `org.nativescript.<folder_name>` for application identifier.